pkg/clustermanager/applier.go

```diff
@@ -173,3 +176,10 @@ func (a Applier) retrierForWait(waitStartTime time.Time) *retrier.Retrier {
 		retrier.WithRetryPolicy(retrier.BackOffPolicy(a.retryBackOff)),
 	)
 }
+
+func (a Applier) retrierForFailureMessage() *retrier.Retrier {
+	return retrier.New(
+		a.waitForFailureMessage,
+		retrier.WithRetryPolicy(retrier.BackOffPolicy(5*retryBackOff)),
```

Comment:
`retryBackOff` wait time is of 1 second. To avoid hitting rate limiting api server I have kept the wait time of 5 seconds between each retry. We can change it if needed.